Shoko.Server/Utilities/AVDumpHelper.cs

```diff
@@ -1,4 +1,4 @@
-﻿#nullable enable
+#nullable enable
```

Comment:
IDE config? Did you change the file format of this? (this almost seems like a change from/to UTF-8 with BOM)

Reply:
Indeed BOM was removed from the file.

Reply:
Included BOM back.